daemons/controld/controld_remote_ra.c

```diff
@@ -23,6 +23,23 @@
 /* The max start timeout before cmd retry */
 #define MAX_START_TIMEOUT_MS 10000
 
+#define cmd_set_flags(cmd_flags, call_for, flags_to_set) do { \
+    cmd_flags = pcmk__set_flags_as(__func__, __LINE__, \
+                                   LOG_TRACE, "remote command", (call_for), (cmd_flags), \
+                                   (flags_to_set), #flags_to_set); \
+        } while (0)
+
+#define cmd_clear_flags(cmd_flags, call_for, flags_to_clear) do { \
+    cmd_flags = pcmk__clear_flags_as(__func__, __LINE__, \
+                                     LOG_TRACE, "remote command", (call_for), (cmd_flags), \
+                                     (flags_to_clear), #flags_to_clear); \
+        } while (0)
```

Comment:
For clear/set macros it's best to give the struct if flags don't need to be set independently, i.e. pass cmd rather than cmd->status. It's easier to read and lets you use other struct fields elsewhere -- e.g. you could use cmd->owner, cmd->rsc_id, or cmd->action rather than passing call_for here

Also the label will start the log line so it's best to capitalize it ("Remote command flags ... for ... set by ...")